extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -133,6 +146,14 @@ export class AmpA4A extends AMP.BaseElement {
     /** @private {!Array<!Promise<!Array<!Promise<?PublicKeyInfoDef>>>>} */
     this.keyInfoSetPromises_ = this.getKeyInfoSets_();
 
+    // TODO(tdrl): Temporary, while we're verifying whether this is an
+    // acceptable solution to the 'Safari on iOS doesn't fetch iframe src
+    // from cache' issue.  See https://github.com/ampproject/amphtml/issues/5614
+    /** @private {?ArrayBuffer} */
+    this.creativeBody_ = null;
+    /** @private {?string} */
+    this.experimentalNonAmpCreativeRenderMethod_ = null;
+
     this.lifecycleReporter_ = getLifecycleReporter(this, 'a4a');
     this.lifecycleReporter_.sendPing('adSlotBuilt');
```

Comment:
Unrelated but let's add a comment that sendPing should be the last action in the constructor.  I'd also add the adSlotBuilt could be somewhat of a misnomer and imply that buildCallback executed as opposed to constructor